### PR TITLE
Changes to improve the use of the module for retrieving Polygons

### PR DESCRIPTION
**Pull request by candrsn**
These changes were helpful to adjust the pyesridump module to build geopackage and spatialite files.

### CODE REVIEW

**Comment by iandees** [comment]
Thanks for suggesting this change. Just a couple changes we need before it can be merged.

File: esridump/dumper.py [hunk 1]
```diff
@@ -154,7 +158,7 @@ def _find_oid_field_name(self, metadata):
         oid_field_name = metadata.get('objectIdField')
         if not oid_field_name:
             for f in metadata['fields']:
-                if f['type'] == 'esriFieldTypeOID':
+                if 'type' in f and f['type'] == 'esriFieldTypeOID':
```
[the remark this answers is not in the document]
Comment (candrsn) [reply esridump/dumper.py]: changed

File: esridump/dumper.py [hunk 2]
```diff
@@ -302,7 +306,11 @@ def __iter__(self):
             # If not, we can still use the `where` argument to paginate
 
             use_oids = True
+            #if self._oidField:
```
[the remark this answers is not in the document]
Comment (candrsn) [reply esridump/dumper.py]: removed

File: .bumpversion.cfg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.7.0
+current_version = 1.7.0.9
```
[the remark this answers is not in the document]
Comment (candrsn) [reply .bumpversion.cfg]: done

**Pull request merged**